ui/apps/platform/src/constants/reportConstants.ts

```diff
@@ -1,17 +1,6 @@
-import { VulnerabilitySeverity } from 'types/cve.proto';
-import { Fixability } from 'types/report.proto';
+import { Fixability } from 'services/ReportsService.types';
 
-type VulnerabilitySeverityLabels = Record<VulnerabilitySeverity, string>;
-
-export const vulnerabilitySeverityLabels: VulnerabilitySeverityLabels = {
-    CRITICAL_VULNERABILITY_SEVERITY: 'Critical',
-    IMPORTANT_VULNERABILITY_SEVERITY: 'Important',
-    MODERATE_VULNERABILITY_SEVERITY: 'Medium',
-    LOW_VULNERABILITY_SEVERITY: 'Low',
-    UNKNOWN_VULNERABILITY_SEVERITY: 'Unknown',
-};
-
-export type FixabilityLabelKey = Exclude<Fixability, 'BOTH' | 'UNSET'>;
+export type FixabilityLabelKey = Exclude<Fixability, 'BOTH'>;
```

Comment:
Why did we remove the `UNSET` option?  Is that value no longer possible in VM Reporting 2.0 ?

Reply:
Good question. I did not see it either in storage (which I assume for v1) nor for v2:
* https://github.com/stackrox/stackrox/blob/master/proto/storage/report_configuration.proto#L50-L54
* https://github.com/stackrox/stackrox/blob/master/proto/api/v2/report_service.proto#L32-L36

Reply:
So I am not sure why it was here:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/types/report.proto.ts#L32